src/styles/utils/reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
According to the checklist, setting the `font-family` in the reset file is not recommended. The `font-family` should be set in a separate typography file or within individual block styles to improve maintainability and separation of concerns.